dev/manual_tests/lib/actions.dart

```diff
@@ -48,8 +48,6 @@ class Memento extends Object with Diagnosticable {
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
     properties.add(StringProperty('name', name));
-    properties.add(FlagProperty('undo', value: undo != null, ifTrue: 'undo'));
```

Comment:
Why get rid of these?

Reply:
Because they were always true, so having them in here just felt like noise as in "of course every Memento has an undo and redo".

Reply:
👍